components/analyticstags.tsx

```diff
@@ -1,7 +1,6 @@
-"use client"
-
-import Script from "next/script"
+import Script from 'next/script'
 
+;('use client')
```

Comment:
this shouldn't be changed I think

Reply:
@jefflinwood and I were just discussing, it is auto lint, I can change, but will probably be changed back